magento/magento2
# Remove `Array.from` polyfill from PrototypeJS 1.7.3 included in `legacy-build.min.js`

PrototypeJS v1.7.3 included in `legacy-build.min.js` breaks native `Array.from` by [forcing](https://github.com/prototypejs/prototype/blob/master/src/prototype/lang/array.js#L101) its own polyfill implementation effectively breaking 3rd party code run on Magento frontend. I

Steps to reproduce
- on Magento frontend run `Array.from(new Set([1, 2, 3]))` snippet
- observe output `[]` empty array

Expected output
- array with 3 elements `[1, 2, 3]`

### Description (*)

Commenting out `Array.from=$A;` from `legacy-build.min.js`

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)
Steps to reproduce
- on Magento frontend run `Array.from(new Set([1, 2, 3]))` snippet
- observe output `[]` empty array

Expected output
- array with 3 elements `[1, 2, 3]`

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38914: Remove `Array.from` polyfill from PrototypeJS 1.7.3 included in `legacy-build.min.js`
